libs/application/templates/no-debt-certificate/src/fields/ConfirmationField/ConfirmationField.css.ts

```diff
@@ -0,0 +1,16 @@
+import { style, globalStyle } from '@vanilla-extract/css'
+
+export const pdfViewer = style({})
+
+globalStyle(`${pdfViewer} canvas`, {
```

Comment:
Just a thought, should we expand the pdf viewer in the ui lib to allow us to set custom width, height, maxwidth and border so there wont be a need for overwriting the global style? Because this is the second application to do this exact style just for these purposes.